testcases/kernel/security/integrity/ima/tests/ima_violations.sh

```diff
@@ -62,7 +62,7 @@ close_file_write()
 
 init()
 {
-	service auditd status > /dev/null 2>&1
+	systemctl status auditd > /dev/null 2>&1
```

Comment:
so instead of assuming one init system, you're changing it to assume a different one.  this isn't really an improvement.